ch-covidcertificate-backend-verifier/ch-covidcertificate-backend-verifier-ws/src/main/java/ch/admin/bag/covidcertificate/backend/verifier/ws/config/WsCloudBaseConfig.java

```diff
@@ -32,10 +29,10 @@ public abstract class WsCloudBaseConfig extends WsBaseConfig {
     @Value("${datasource.leakDetectionThreshold:0}")
```

Comment:
ditto unused properties. ok that they are not being applied to the datasource config anymore